<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>socialnetworkui</class>
 <widget class="QMainWindow" name="socialnetworkui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>socialnetworkui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFriend">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginLine">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameProfile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Friends</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>83</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Recent Posts</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Friend Suggest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FriendSuggestions">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Suggest</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>161</width>
      <height>400</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>159</width>
       <height>129</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="FriendList"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>151</width>
      <height>400</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>129</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QListWidget" name="FriendSuggestList"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>165</width>
      <height>216</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>163</width>
       <height>214</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="TablePosts">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <zorder>AddFriend</zorder>
   <zorder>HomeButton</zorder>
   <zorder>LoginLine</zorder>
   <zorder>NameProfile</zorder>
   <zorder>label_2</zorder>
   <zorder>scrollArea</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>scrollArea_2</zorder>
   <zorder>FriendSuggestions</zorder>
   <zorder>LoginButton</zorder>
   <zorder>scrollArea_3</zorder>
   <zorder>backButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
